package.json

```diff
@@ -33,8 +33,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*.test.js",
-    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --runInBand --colors ./src/integration/pages/**/*.test.js",
+    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --runInBand --colors ./src/integration/pages/**/*",
```

Comment:
Because `(npm run start...)` is backgrounded, it isn't terminated when `test:integration` completes. This prevents later run start's/dev's, with the following error:
 
```
Error: bind EADDRINUSE null:7080
    at listenOnMasterHandle (net.js:1380:18)
    at rr (internal/cluster/child.js:132:12)
    at Worker.<anonymous> (internal/cluster/child.js:99:7)
    at process.onInternalMessage (internal/cluster/utils.js:47:8)
    at process.emit (events.js:322:22)
    at emit (internal/child_process.js:876:12)
    at processTicksAndRejections (internal/process/task_queues.js:85:21)
```

Could we utilise `killApp` or write a `runTestIntegration` script that manages the later part of this command?